include/mlir/Dialect/Linalg/Transforms/LinalgTransformPatterns.td

```diff
@@ -57,9 +57,10 @@ class TileAndFuseLinalgOp<
 // In the future, tile sizes should be derived from op properties + machine
 // description but we do not need to wait on this to start having useful
 // patterns.
```

Comment:
Can you plz describe the permutation parameter?

Reply:
Added some further description of the paremeter